### ISSUE TITLE
Button doesn't trigger action

### ISSUE THREAD

**Issue by Hellstormer00** [Toxblh/MTMR#155]
`{
    "type": "staticButton",
    "align": "left",
    "width": 44,
    "image": {
      "base64": "D0...BlaBla"
  },
  "action": "appleScript",
  "actionAppleScript": {
    "inline":
      "tell application \"Google Chrome\"\rset minimized of window 1 to true\rend tell"
    },
},`

I gave MTMR all permissions it needed but nevertheless nothing happens when I try pressing this Button. When i run the script in the Apple Script editor everything works fine and Chrome get's minimized. Thank your for your support :)

**Comment by Toxblh** [comment]
So is it work or not for you?

**Comment by Toxblh** [comment]
I tried your code - and it's work fine for me. 
When I click - Chrome is minimised.

**Comment by Toxblh** [comment]
If not let's try to check the tick 
![image](https://user-images.githubusercontent.com/2198153/54725882-93cedb00-4b68-11e9-9977-df270e4ccdff.png)

**Comment by Hellstormer00** [comment]
Thanks so much for your response but: 
When i check my Privacy --> automation settings the only requested permission i can check is Safari for MTMR. This is the only checkbox. There isn't anything else

**Comment by Toxblh** [comment]
Let's check some theory.
Could you re-tick access in Accessibility, restart MTMR and try press button again

**Comment by Hellstormer00** [comment]
Yeah it didn't work :(

**Comment by Hellstormer00** [comment]
I also tried "filePath" to the script but it also doesnt work

**Comment by Toxblh** [comment]
So the main point - MacOS didn't ask or ask and get discard. And you can't reask again. Need to find how to get re-ask about access to google chrome via apple script.

**Comment by Hellstormer00** [comment]
hmm. Im pretty sure it has nothing to do with Chrome. Because when i replace "Google Chrome" with "Safari" also nothing happens

**Comment by Hellstormer00** [comment]
No matter what kind of script i write it doesn't work. Maybe somethings wrong with the code:
<img width="1440" alt="Screenshot 2019-03-21 at 16 08 02" src="https://user-images.githubusercontent.com/48771252/54762084-b3f7ac00-4bf3-11e9-9e13-9432f44d2027.png">

**Comment by Toxblh** [comment]
Ok, let's try to understand that so happen.
Could you open console.app, find `mtmr` next
try few times press on you button for chrome and safari next
select all messages cmd+a, cmd+c and cmd+v here

**Comment by matoelorriaga** [comment]
You shouldn’t be using comments in json file, remove them an it should work

**Comment by Toxblh** [comment]
@matoelorriaga it's ok because comments will delete on pre-parse

**Comment by Hellstormer00** [comment]
error	18:09:47.909842 +0100	MTMR	Unable to load Info.plist exceptions (eGPUOverrides)
default	18:09:57.910038 +0100	MTMR	LSExceptions shared instance invalidated for timeout.
error	18:10:30.869498 +0100	MTMR	Unable to load Info.plist exceptions (eGPUOverrides)
default	18:10:40.869647 +0100	MTMR	LSExceptions shared instance invalidated for timeout.

I hope these are the ones your are looking for

**Comment by Toxblh** [comment]
Thanks I will see how to fix it

**Comment by Toxblh** [comment]
lol
https://stackoverflow.com/questions/52507950/unable-to-load-info-plist-error-xcode-10-under-macos-10-14-mojave

The super strange bug. I already provide all descriptions for everything...
Will find :)

**Comment by Toxblh** [comment]
Short summary
This doesn't work because you can't get a window with the security request 🤦‍♂️ 
I will find how to push it (and don't affect who want to discard this)

**Comment by Toxblh** [comment]
HardCode way - reset all access for programs for AppleEvents 
`tccutil reset AppleEvents`

src: https://www.felix-schwarz.org/blog/2018/08/new-apple-event-apis-in-macos-mojave

**Comment by Hellstormer00** [comment]
Yeah i suspected the bug to be something in that direction. Thank you so much for your help.

**Comment by Hellstormer00** [comment]
But it still doesn't work...

**Comment by Toxblh** [comment]
didn't check yet :) GitHub!

**Comment by Toxblh** [comment]
try 0.20.1 I change to the old build system as in manuals

**Comment by Hellstormer00** [comment]
🙄Still nothing i could try reticking accessibility but im pretty sure it will not help

**Comment by Toxblh** [comment]
I have a stupid idea - maybe restart Macbook?

**Comment by Toxblh** [comment]
delete mtmr
restart 
and install again

**Comment by Hellstormer00** [comment]
Ok you know what i dont want to annoy you it's probably my fault i don't want to waste your time

**Comment by Toxblh** [comment]
don't worry, it's the famous bug in MacOS and would be glad to find a solution to fix it in future

**Comment by Hellstormer00** [comment]
ok so no reinstalling and restarting didnt wirk

**Comment by Hellstormer00** [comment]
error	00:48:02.647636 +0100	MTMR	nw_protocol_boringssl_get_output_frames(1301) <private>[0x7f927cf26d30] get output frames failed, state 8196
error	00:48:02.647715 +0100	MTMR	nw_protocol_boringssl_get_output_frames(1301) <private>[0x7f927cf26d30] get output frames failed, state 8196
error	00:48:02.648167 +0100	MTMR	TIC Read Status [1:0x0]: 1:57
error	00:48:02.648201 +0100	MTMR	TIC Read Status [1:0x0]: 1:57
default	00:48:14.187390 +0100	MTMR	Failed to resolve bookmark 7f927d094800 with error: Error Domain=NSCocoaErrorDomain Code=4 "The file doesn’t exist."
default	00:49:04.063888 +0100	MTMR	Failed to resolve bookmark 7f927d094800 with error: Error Domain=NSCocoaErrorDomain Code=4 "The file doesn’t exist."
default	00:50:13.222869 +0100	MTMR	Failed to resolve bookmark 7f927d094800 with error: Error Domain=NSCocoaErrorDomain Code=4 "The file doesn’t exist."
default	00:50:18.660889 +0100	MTMR	27366555: RECEIVED OUT-OF-SEQUENCE NOTIFICATION: 114 vs 133, 512, <private>
default	00:50:18.670382 +0100	MTMR	Failed to resolve bookmark 7f927d094800 with error: Error Domain=NSCocoaErrorDomain Code=4 "The file doesn’t exist."
default	00:50:21.133990 +0100	MTMR	Failed to resolve bookmark 7f927d094800 with error: Error Domain=NSCocoaErrorDomain Code=4 "The file doesn’t exist."

**Comment by Hellstormer00** [comment]
no more Info.plist...

**Comment by msanchez-eutech** [comment]
Same problem here but i found a workaround by using bash to execute the apple script for me.

```json
{
          "type": "staticButton",
          "action": "shellScript",
          "executablePath": "/bin/bash",
          "shellArguments": [
            "-c", "osascript -e 'tell application \"Microsoft Teams\" to activate'"
          ],
          "title": "Teams",
}
```
